app/views/users/rules_of_use/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<% title t('titles.registrations.new') %>
+
+<h1><%= t('titles.rules_of_use') %></h1>
+
+<p>
+<%== t('users.rules_of_use.overview', link: new_window_link_to(t('titles.rules_of_use'),
+                                                              "http://localhost:4000/policy/rules-of-use/")) %>
```

Comment:
😬 the comment was resolved but the localhost URL is still there.

Also, I noticed the `<%==` ... what if we updated the key to be `overview_html` so i18n would mark things as html-safe for us?